helpers.go

```diff
@@ -358,3 +364,23 @@ func removeDuplicates(elements []string) []string {
 
 	return res
 }
+
+// format and print given images to match `docker images` output
+func formatAndPrint(images []types.ImageSummary) {
+
+	writer := tabwriter.NewWriter(os.Stdout, 20, 1, 3, ' ', 0)
+	fmt.Fprintln(writer, "REPOSITORY\tTAG\tIMAGE ID\tCREATED\tSIZE")
+
+	for _, img := range images {
+		for _, repoTag := range img.RepoTags {
+			repo := strings.Split(repoTag, ":")[0]
+			tag := strings.Split(repoTag, ":")[1]
+			truncID := stringid.TruncateID(img.ID)
+			createdSince := units.HumanDuration(time.Now().UTC().Sub(time.Unix(img.Created, 0))) + " ago"
+			hSize := units.HumanSize(float64(img.Size))
+
+			fmt.Fprintln(writer, repo+"\t"+tag+"\t"+truncID+"\t"+createdSince+"\t"+hSize)
+		}
+	}
```

Comment:
Please use `fmt.Fprintf(...)` instead of `fmt.Println(...)`. Printf is generally faster and more explicit (e.g., it's typed) making the code easier to read and to maintain.